src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
+import { Movie } from '../../types/movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    imgUrl, title, description, imdbUrl, imdbId,
+  } = movie;
+
+  return (
+    <div
+      className="card"
+      data-cy="card"
+      key={imdbId}
```

Comment:
better set a key when you call your `MovieCard` from `MoviesList` 😉

Reply:
fixed